### ISSUE TITLE
A11y_.NETCore3.0(WPF )_WPFConcepts_MenuSample_Screenreader: Screen reader is announcing the wrong name of the drop down menu item "Project"

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1466]
Copied from VSO: [956407](https://devdiv.visualstudio.com/DevDiv/_queries/edit/956407/)
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:

Application Name: .NET Core 3.0 WPF

.NET Core SDK version:  3.0.100-preview8-013442
Windows Version: Windows10

Screen Reader Vesion : Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link.

Repro Steps:

1) Run exe file for "Concepts" from the build app folder provided. 
2) Main Window for "Concepts" solution will appear. 
3) Select "Samples" tab item. 
4) From right navigation under "Controls" select "Menu Sample". 
5) Navigate to the preview tab and the to Project menu item.
6) start screen reader and observe the screen reader announcement.

Actual:

Screen reader is announcing the wrong name of the menu item "Project" and sub menu item inside "Project".

Expected:

Screen reader should announce proper name of the controls.

User Impact:

Screen reader user may not identify the control as name is incorrect.

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS1.3.1

https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by arpitmathur** [comment]
This is a sample app issue and not a framework issue.

Removing the TextBlock from the MenuItem Header resolves the issue.

If I define the MenuItem as :

```
 <MenuItem>
   <MenuItem.Header>_Project</MenuItem.Header>
 <MenuItem>
```



Now, on navigating to this menu item, narrator reads "Project Menu item 1 of 4, Menu Key P"
